sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/Transforms.java

```diff
@@ -265,7 +267,8 @@ private Transforms() {
                 break;
             case TIME:
                 value = new FormField<>(setDefaultConfidenceValue(fieldValue.getConfidence()), labelText,
-                    key, fieldValue.getValueTime(), valueText);
+                    key, LocalTime.parse(fieldValue.getValueTime(), DateTimeFormatter.ofPattern("HH:mm:ss")),
```

Comment:
if getValueTime() returns null will this still work? I've seen inputs where the time was extracted from the receipt as not a real time, e.g. '36:54:47 PM', and then although the field type is set to "time" it returns null for valueTime.

Reply:
How will that scenario look in the swagger? do you have an example?

Reply:
Will send to you guys in chat since the example is a personal receipt

Reply:
Updated to add a null check before parsing, thanks Krista!